stroeer/core/v1/core_article_service.proto

```diff
@@ -71,8 +73,8 @@ message GetArticleRequest {
  */
 message BatchGetArticlesRequest {
   repeated int64 ids = 1;
+  Legal legal = 2;
```

Comment:
do we filter elements only in the implementation or are unpublished articles also removed from batch responses? this should be clear in the documentation and the filter options